database/seeding.js

```diff
@@ -0,0 +1,58 @@
+const fs = require('fs');
+const path = require('path');
+
+function getRandomName() {
+  return Math.floor(Math.random() * 300);
+}
+
+const firstName = ['Octavio', 'Javier', 'Davin', 'Grant', 'Salvatore', 'Ayaan', 'Dorian', 'Clay', 'Raymond', 'Guillermo', 'Cole', 'Curtis', 'Ace', 'Brian', 'Rex', 'Charles', 'Hezekiah',  'Clarence', 'Joe', 'Brenden', 'Ronin', 'Mark', 'Matteo', 'Davian', 'Fisher', 'Brodie', 'Adam', 'Mike', 'Alexander', 'Phillip', 'Trystan', 'Judah', 'Mauricio', 'Collin', 'Luke', 'Clayton', 'Killian', 'Layton', 'Lee', 'Mateo', 'Noe', 'Kylan', 'Simon', 'Alberto', 'Julian', 'Ezra', 'Alan', 'Kian', 'Aiden', 'Abraham', 'Moises', 'Pierce', 'Quinn', 'Bernard', 'Keyon', 'Byron', 'Duncan', 'Drew', 'Dwayne', 'Rayan', 'Darren', 'Brennen', 'Orion', 'Soren', 'Howard', 'Jonah', 'Samir', 'Shamar',
```

Comment:
Looks good, but check out the faker library! You can easily generate all sorts of data